sdk/storage/azure-storage-file-datalake/tests/test_file_async.py

```diff
@@ -430,11 +505,14 @@ async def test_upload_data_async(self, datalake_storage_account_name, datalake_s
         await file_client.upload_data(data, overwrite=True, max_concurrency=5)
 
         downloaded_data = await (await file_client.download_file()).readall()
-        self.assertEqual(data, downloaded_data)
+        assert data == downloaded_data
 
     @pytest.mark.live_test_only
     @DataLakePreparer()
-    async def test_upload_data_to_existing_file_async(self, datalake_storage_account_name, datalake_storage_account_key):
+    async def test_upload_data_to_existing_file(self, **kwargs):
```

Comment:
This test and the next 2 are not live only in sync. This test does not use any concurrency so good to record. The others have concurrency but a blob size of 100 so again, it's doing nothing. Can you remove any concurrency here and record these?
Can you also update sync to remove concurrency from same tests?